tracer.go

```diff
@@ -21,10 +21,12 @@ import (
 	"go.opentelemetry.io/otel/trace"
 )
 
+type contextKey int8
+
 const (
 	tracerName          = "github.com/exaring/otelpgx"
 	meterName           = "github.com/exaring/otelpgx"
-	startTimeCtxKey     = "otelpgxStartTime"
+	startTimeCtxKey     = contextKey(0)
```

Comment:
Thanks for the suggestion! That makes sense!
However: since we're touching this, it would probably make sense to replace it with a `struct{}`, so that it [uses no memory](https://dave.cheney.net/2014/03/25/the-empty-struct)?